src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+public class Ball {
+    private final String color;
```

Comment:
Consider using an access modifier for the class fields 'color' and 'number'. If these fields are not meant to be modified outside of the class, 'private' would be appropriate.